managers/backup-manager/BackupService.js

```diff
@@ -249,6 +249,26 @@ class BackupService extends BaseDirectorService {
       });
   }
 
+  getLastBackup(tenant_id, instance_guid, noCache) {
```

Comment:
from which invocation is `noCache` set? If this is never going to be used in V2, recommed it to be rather removed.

Reply:
Not currently used but this is an optimisation we can have in cases we need to avoid requests to agent.